WooCommerce/Classes/ViewRelated/Notifications/ReviewsViewModel.swift

```diff
@@ -105,9 +105,11 @@ extension ReviewsViewModel {
         let action = ProductReviewAction.synchronizeProductReviews(siteID: siteID, pageNumber: 1, pageSize: 25) { error in
             if let error = error {
                 DDLogError("⛔️ Error synchronizing reviews: \(error)")
+                ServiceLocator.analytics.track(.reviewsListLoadFailed,
+                                               withError: error)
             } else {
-                //TODO. What event must be sent here?
-                //ServiceLocator.analytics.track(.notificationListLoaded)
+                ServiceLocator.analytics.track(.reviewsListLoaded,
+                                               withProperties: ["is_loading_more": false])
```

Comment:
just wondering, is `is_loading_more` set to true under some other conditions?

Reply:
Yes, it should be true when loading new pages of data (see p99K0U-1Ln-p2 for reference)

Reply:
`is_loading_more` seems to be only set to false in the PR, is there another code path or condition when it might be true?

Reply:
Not yet. When implementing pagination ( #1267 ) it would be set to true